cloud-architecture/custom-apps-and-consolidation/oracle-db-consolidation/db-discovery/discovery-utility/files/oradb_discovery_v3_6_11_19.sql

```diff
@@ -0,0 +1,2758 @@
+--# Copyright (c) 2023, Oracle and/or its affiliates.
+--# Licensed under the Universal Permissive License v1.0 as shown at https://oss.oracle.com/licenses/upl.
+--# Purpose: To capture database details for Oracle database 11g onwards 
+--# This script is to be executed at the discovery phase before planning for migration/upgrade
+--# @author: Ananda Ghosh Dastidar
+
+
+clear screen;
+set echo off;
+set feedback off;
+set verify off;
+set pagesize 10000;
+SET linesize 32767;
+set serveroutput on size 1000000;
+
+variable gv_multi NUMBER;
+variable gv_version NUMBER;
+variable gv_dbinfosql REFCURSOR;
+variable gv_dbname varchar2(30);
+define gv_script_version='3.6';
+
+--Added for centralized discovery
+variable gv_pdbconid NUMBER;
+col pdbconid new_value v_pdbconid;
+
+
+--# Validation Block 1: Checking eligible DB version to run this script
+PROMPT
+DECLARE
+v_version number;
+BEGIN
+EXECUTE IMMEDIATE 'select replace(version,''.'') dbversion from sys.v_$instance' INTO v_version;
+IF v_version < 110000 THEN
+DBMS_OUTPUT.put_line('PROMPT Error!! This script is intended to execute on oracle database 11g or higher version.');
+:gv_version := 1;
+END IF;
+EXCEPTION WHEN OTHERS THEN
+DBMS_OUTPUT.PUT_LINE ('PROMPT Error!! '||SQLCODE ||' : '||SUBSTR(SQLERRM,1,64));
+:gv_version := 1;
+END;
+/
+PROMPT
+set termout off;
+SPOOL version_not_matched.sql
+define _editor=vi
+DECLARE
+	PROCEDURE delsqlfile IS
+	v_os_platform varchar2(200);
+	BEGIN
+	EXECUTE IMMEDIATE 'SELECT lower(PLATFORM_NAME) FROM sys.v_$database' INTO v_os_platform;
+	IF(v_os_platform LIKE  '%window%') THEN
+	dbms_output.put_line('host del /Q '||'version_not_matched.sql');
+	END IF;
+	IF( (v_os_platform LIKE  '%solaris%') OR (v_os_platform like  '%linux%') OR (v_os_platform like  '%aix%') ) THEN
+	dbms_output.put_line('host rm -rf '||'version_not_matched.sql');
+	END IF;
+	END;
+
+BEGIN
+IF :gv_version = 1 THEN
+DBMS_OUTPUT.put_line('PROMPT Error!! This script is intended to execute on oracle database 11g or higher version.');
+delsqlfile;
+dbms_output.put_line('exit;');
+ELSE
+delsqlfile;
+END IF;
+END;
+/
+SPOOL OFF
+@version_not_matched.sql

```

Comment:
This file is not included. Where to find it?

Reply:
Submitted with required changes.